app/src/main/java/com/example/concordia_campus_guide/Models/ListOfCoordinates.java

```diff
@@ -0,0 +1,16 @@
+package com.example.concordia_campus_guide.Models;
+
+import java.util.List;
+
+public class ListOfCoordinates {
```

Comment:
Did you create this class to be able to read from the geojson file for coordinates ?

Reply:
yes, the orm gave issues when we had List<Coordinates> in the Building class